packages/picongpu/package.py

```diff
@@ -84,8 +85,9 @@ class Picongpu(Package):
     depends_on('adios@1.13.1:,develop', when='+adios')
     depends_on('isaac@1.4.0', when='@:0.4.3 +isaac')
     depends_on('isaac-server@1.4.0', type='run', when='@:0.4.3 +isaac')
-    depends_on('isaac@1.5.0,develop', when='@0.5.0: +isaac')
-    depends_on('isaac-server@1.5.0,develop', type='run', when='@0.5.0: +isaac')
+    depends_on('isaac@1.5.2,develop', when='@0.5.0: +isaac')
+    depends_on('isaac-server@1.5.2,develop', type='run', when='@0.5.0: +isaac')
+    depends_on('openpmd-api@dev', type='run', when='@develop')
```

Comment:
This is a regular lib dependency:
```suggestion
    depends_on('openpmd-api@dev', when='@develop')
```

You can also add a variant for it as we have for HDF5, so you can control the dependencies it pulls a bit better:
```py
    variant('openpmd', default=True, description='Enable openPMD I/O')
```
Then this line will read:
```suggestion
    depends_on('openpmd-api@dev', when='@develop +openPMD')
```

Reply:
fixed